### PR TITLE
devtools storage cleanups

### PR DESCRIPTION
**Pull request by samthor**
Taken over from #958, which only removed the image from one page.

While I was here, I've:
- removed Application Cache (dropped in m85)
- made Web SQL disuse more clear

### CODE REVIEW

File: site/en/docs/devtools/storage/websql/index.md [hunk 1]
```diff
@@ -10,37 +10,30 @@ description: "How to view Web SQL data from the Application panel of Chrome DevT
...
 
-1.  Click the **Sources** tab to open the **Application** panel. The **Manifest** pane usually opens by
-    default.
-
-    {% Img src="image/admin/5z7Wz6UKaPOlgwmsaDMm.png", alt="The Manifest pane.", width="800", height="619" %}
-
-    **Figure 1**. The Manifest pane.
-
-2.  Expand the **Web SQL** section to view databases and tables. In **Figure 2** below
+1.  Click the **Sources** tab to open the **Application** panel.Expand the **Web SQL** section to view databases and tables. In **Figure 1** below
```
Comment (jpmedley) [inline site/en/docs/devtools/storage/websql/index.md:21]:
```suggestion
1.  Click the **Sources** tab to open the **Application** panel. Expand the **Web SQL** section to view databases and tables. In **Figure 1** below
```

**Review by jpmedley — CHANGES_REQUESTED** [review]
Andy's OOO, so I picked this up.

File: site/en/docs/devtools/storage/cache/index.md [hunk 2]
```diff
@@ -88,13 +82,13 @@ column of the **Network Log** has the information you're looking for. See [Log n
...
 
-    **Figure 9**. The **Cache Storage** checkbox.
+    **Figure 8**. The **Cache Storage** checkbox.
 
 3.  Click **Clear site data**.
 
     {% Img src="image/admin/C0MQhJ0YhFIuKudvcghG.png", alt="The Clear Site Data button.", width="800", height="572" %}
 
-    **Figure 10**. The **Clear Site Data** button.
+    **Figure 0**. The **Clear Site Data** button.
```
Comment (chybie) [inline site/en/docs/devtools/storage/cache/index.md:91]: Figure 9

**Review by chybie — APPROVED** [review]
LGTM with one minor typo

**Pull request merged**